### DISCUSSION TITLE
Scratchpad windows not working

### DISCUSSION THREAD

**Discussion by hankleonard** [i3/i3#4526] [Q&A]
Hello all,

I've been following a tutorial I found on YouTube for establishing "dropdown" windows through the use of scratchpad. My intended use is to be able to launch a terminal, ranger, cmus, etc. instance and have it reappear/disappear with the keybinding. Despite following the tutorial the code in the config file seems not to be working. See the excerpt from the file below.

```
$term = xfce4-terminal

for_window [instance="ranger"] floating enable
for_window [instance="ranger"] resize set 625 400
for_window [instance="ranger"] move scratchpad
for_window [instance="ranger"] border pixel 5
exec --no-startup-id $term -n ranger -e ranger

bindsym $mod+p [instance="ranger"] scratchpad show; [instance="ranger"] move position center 
```

Thank you for your help!

**Comment by haddadnidal** [comment]
hi did you resolve the problem ? if not here is a code sample from my `i3/config` file for the dropdown.

`for_Window [instance ="scratch"] floating enable`
 `for_Window [instance ="scratch"] resize set 650 400`
 `for_Window [instance ="scratch"] move absolute position center`
  `for_Window [instance ="scratch"] move scratchpad; [instance ="scratch"] scratchpad show`
 `bindsym $mod+u [instance ="scratch"]  scratchpad show`
 `bindsym $mod+Shift+u exec --no-startup-id kitty --name scratch &`
